pom.xml

```diff
@@ -270,6 +270,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ]]>
                     </inlineHeader>
+                    <headerSections>
+                        <headerSection>
+                            <key>YEAR</key>
+                            <defaultValue>2019</defaultValue>
+                            <ensureMatch>[0-9-]+</ensureMatch>
+                        </headerSection>
+                    </headerSections>
                     <mapping>
                         <pom.xml>XML_STYLE</pom.xml>
```

Comment:
I think you could remove both pom.xml and cfg mappings. (pom should already be in XML format and there are no .cfg files in the project)